lessons/Algebra2/periodic1-exploring-rides/langs/en-us/index.adoc

```diff
@@ -104,173 +99,238 @@ This worksheet had you create a *new type of data visualization*, called a @voca
 
 @slidebreak
 
-In this example, we know the Ferris wheel keeps turning, generating data "in between" the times represented in our scatter plot. Connecting those dots with the line gives us an *approximation* of what data is missing between those times, and helps us see the shape of the relationship.
+In this example, we might only have data for the Ferris Wheel every 5 minutes, but we know it keeps turning, generating data "in between" the times represented in our scatter plot and there are an infinite number of points _between_ those times. We also know that the Ferris Wheel existed at every one of those points, meaning there was `altitude` data to be gathered at all times.
```

Comment:
@schanzer Noticed that capitalization of Ferris Wheel was inconsistent throughout this lesson. Have streamlined to Ferris wheel. Feel free to do a find and replace if you want.

Reply:
Find-and-replace complete